java/src/jmri/managers/AbstractTurnoutManager.java

```diff
@@ -216,7 +216,9 @@ public boolean isControlTypeSupported(@Nonnull String systemName) {
      *
      * @param systemName system name.
      * @param userName username.
```

Comment:
These two should be removed since they are duplicated. Javadoc check complained about these in CI.